src/Nest/QueryDsl/Compound/Indices/IndicesQuery.cs

```diff
@@ -1,74 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using Newtonsoft.Json;
-
-namespace Nest
-{
-	[JsonObject(MemberSerialization = MemberSerialization.OptIn)]
-	[JsonConverter(typeof(ReadAsTypeJsonConverter<IndicesQueryDescriptor<object>>))]
-	[Obsolete("Deprecated. You can specify _index on the query to target specific indices")]
-	public interface IIndicesQuery : IQuery
```

Comment:
I think this is a good candidate for an example in the documentation i.e. combining a query on `_index` with other queries to target a specific index  in a request that targets multiple indices